src/clj_kondo/impl/analyzer/test.clj

```diff
@@ -40,3 +40,10 @@
                                                                     is-expr
                                                                     args)))]
     (common/analyze-expression** ctx new-node)))
+
+(defn testing-hook [{:keys [node]}]
+  (let [[testing testing-str & children] (:children node)
+        new-node (assoc node :children
+                        (list* (assoc-in testing [:context :clojure.test :testing-str] (utils/sexpr testing-str))
```

Comment:
Should we consider `cljs.test` as well?

Reply:
Do you mean the keyword? I think we can use the same keyword for both dialects?

Reply:
I'm ok with that